UI/src/sass/enquiry-build.scss

```diff
@@ -11,3 +11,8 @@
     width: 25%;
   }
 }
+
+.pii-warning {
+  margin-bottom: 10px;
```

Comment:
Using "em" units will make the margin adjust proportionally based on the font size of the element or the root element, respectively. This ensures that the margin will be consistent across different devices and screen sizes.

For example:
```
.pii-warning {
  margin-bottom: 0.53em;
  margin-top: 0.53em;
}
```

Reply:
I have updated this as suggested, although I don't really think it was worth raising or doing since I think that px is used a lot more than em in the existing .scss.